docs/data-sources/supported-sources.md

```diff
@@ -0,0 +1,65 @@
+  [Phabricator](https://github.com/chaoss/grimoirelab-sirmordred#phabricator-),
+  [Redmine](https://github.com/chaoss/grimoirelab-sirmordred#redmine-),
+  [Pagure](https://github.com/chaoss/grimoirelab-sirmordred#pagure-)
+- **Code Review**: [Gerrit](https://github.com/chaoss/grimoirelab-sirmordred#gerrit-)
+- **Containers/Packaging**:
+  [DockerHub](https://github.com/chaoss/grimoirelab-sirmordred#dockerhub-),
+  [crates](https://github.com/chaoss/grimoirelab-sirmordred#crates-)
+- **Continuous Integration**:
+  [Jenkins](https://github.com/chaoss/grimoirelab-sirmordred#jenkins-),
+  [Puppetforge](https://github.com/chaoss/grimoirelab-sirmordred#puppetforge-),
+  [functest](https://github.com/chaoss/grimoirelab-sirmordred#functest-)
+- **Wiki**:
+  [Confluence](https://github.com/chaoss/grimoirelab-sirmordred#confluence-),
+  [MediaWiki](https://github.com/chaoss/grimoirelab-sirmordred#mediawiki-)
+- **Questions and Answer Forums**:
+  [Askbot](https://github.com/chaoss/grimoirelab-sirmordred#askbot-),
+  [Discourse](https://github.com/chaoss/grimoirelab-sirmordred#discourse-),
+  [Stack
+  Exchange](https://github.com/chaoss/grimoirelab-sirmordred#stackexchange-),
+  [KitSune](https://github.com/chaoss/grimoirelab-sirmordred#kitsune-)
+- **Mailing Lists**:
+  [Pipermail](https://github.com/chaoss/grimoirelab-sirmordred#pipermail-),
+  [NNTP](https://github.com/chaoss/grimoirelab-sirmordred#nntp-),
+  [HyperKitty](https://github.com/chaoss/grimoirelab-sirmordred#hyperkitty-),
+  [Groupsio](https://github.com/chaoss/grimoirelab-sirmordred#groupsio-),
+  [mbox](https://github.com/chaoss/grimoirelab-sirmordred#mbox-)
+- **Chats**: [Slack](https://github.com/chaoss/grimoirelab-sirmordred#slack-),
+  [Telegram](https://github.com/chaoss/grimoirelab-sirmordred#telegram-),
+  [Supybot](https://github.com/chaoss/grimoirelab-sirmordred#supybot-),
+  [RocketChat](https://github.com/chaoss/grimoirelab-sirmordred#rocketchat-),
+  [Mattermost](https://github.com/chaoss/grimoirelab-sirmordred#mattermost-),
+  [Gitter](https://github.com/chaoss/grimoirelab-sirmordred#gitter-)
+- **Meetings/Events**:
+  [Meetup](https://github.com/chaoss/grimoirelab-sirmordred#meetup-),
+  [Mozillaclub](https://github.com/chaoss/grimoirelab-sirmordred#mozillaclub-),
+  [remo](https://github.com/chaoss/grimoirelab-sirmordred#remo-)
+- **Social Networks**:
+  [Twitter](https://github.com/chaoss/grimoirelab-sirmordred#twitter-)
+- **Others**: [RSS](https://github.com/chaoss/grimoirelab-sirmordred#rss-),
+  [Cocom](https://github.com/chaoss/grimoirelab-sirmordred#cocom-),
+  [Colic](https://github.com/chaoss/grimoirelab-sirmordred#colic-),
+  [Weblate](https://github.com/chaoss/grimoirelab-sirmordred#weblate-),
+  [google_hits](https://github.com/chaoss/grimoirelab-sirmordred#google_hits-),
+  [Dockerdeps](https://github.com/chaoss/grimoirelab-sirmordred#dockerdeps-),
+  [dockersmells](https://github.com/chaoss/grimoirelab-sirmordred#dockersmells-)
```

Comment:
Can you add one more para, after this, adding a reference to https://github.com/chaoss/grimoirelab/blob/master/community_components.md#other-data-sources?